code/lib/preview-api/src/modules/store/csf/prepareStory.test.ts

```diff
@@ -606,12 +612,12 @@ describe('prepareStory', () => {
         { render: renderMock }
       );
 
-      const context = firstStory.prepareContext({
+      const context = prepareContext({
         args: firstStory.initialArgs,
         hooks: new HooksContext(),
         ...firstStory,
       } as any);
-      firstStory.unboundStoryFn(context);
+      firstStory.unboundStoryFn(context as any);
```

Comment:
Will this get more typesafe when you make the changes to `@storybook/csf`?

Reply:
I'm confused about the types TBH, but hopefully.

Reply:
I think it's good now, @kasperpeulen gave me some advice.